Breeze/src/Breeze.Wallet/Controllers/WalletController.cs

```diff
@@ -94,24 +93,24 @@ public IActionResult Recover([FromBody]WalletRecoveryRequest walletRecovery)
 
             try
             {
-                var wallet = this.walletWrapper.Recover(walletRecovery.Password, walletRecovery.FolderPath, walletRecovery.Name, walletRecovery.Network, walletRecovery.Mnemonic);
-                return this.Json(wallet);
+                this.walletWrapper.Recover(walletRecovery.Password, walletRecovery.FolderPath, walletRecovery.Name, walletRecovery.Network, walletRecovery.Mnemonic);
+	            return this.Json(new SuccessModel {Success = true});
 
             }
             catch (FileNotFoundException e)
             {
 				// indicates that this wallet does not exist
-				return ErrorHelpers.BuildErrorResponse(HttpStatusCode.NotFound, "Wallet not found.", e.ToString());				
+				return ErrorHelpers.BuildErrorResponse(HttpStatusCode.Conflict, "This wallet already exists.", e.ToString());	
```

Comment:
Yes. This is what you intended, you just mixed up the recover and load error check:(https://github.com/stratisproject/Breeze/pull/21#discussion_r110545985)